test/lib/render-helpers.js

```diff
@@ -186,7 +186,11 @@ export async function integrationTestRender(extendedRenderOptions) {
     ...renderOptions
   } = extendedRenderOptions;
 
-  const store = await setupInitialStore(preloadedState, backgroundConnection, {
+  connectToBackground(null, backgroundConnection, () => {
```

Comment:
`setupInitialStore` no longer calls `connectToBackground` (since that is a job of other code, not a function named `setupInitialStore`). `setupInitialStore` is still poorly named, since it does way more than that, but I can't just go around fixing all these sorts of things at once!